config/routes.rb

```diff
@@ -10,8 +10,10 @@
 
   namespace :api do
     namespace :v0 do
-      resources :markets, only: [:index, :show]
-      resources :vendors, only: [:index, :show, :create]
+      resources :markets, only: [:index, :show] do
+        resources :vendors, only: [:index]
```

Comment:
I see you did this route  with the feedback we received during our project check-in